### PR TITLE
CFE-636 Test that methods can be called by classic arrays

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @olehermanse can review this?

**Comment by olehermanse** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/1883/)

https://ci.cfengine.com/job/pr-pipeline/1883/

File: tests/acceptance/21_methods/call_methods_using_array_expansion.cf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#######################################################
+#
+# Redmine 6369 - make sure that arrays don't accumulate
+#
```
Comment (craigcomstock) [inline tests/acceptance/21_methods/call_methods_using_array_expansion.cf]: Redmine??? I know that's an old bug system but why mentioned here? Shouldn't it be CFE-636 mentioned here?

Comment (olehermanse) [reply tests/acceptance/21_methods/call_methods_using_array_expansion.cf]: @craigcomstock @nickanderson Feel free to fix this in a follow-up.

**Pull request merged**